src/client.jsx

```diff
@@ -44,7 +44,7 @@ window.errorReporter = ErrorReporter.getInstance({
 
 configureTracker({
   listen: browserHistory.listen,
-  gaTrackingId: config.gaTrackingId,
+  gaTrackingId: window.location.host ? config.gaTrackingId : '',
```

Comment:
Host er jo alltid der? Hvorfor gjøre dette?

Reply:
Det er for å ikke ha med det når vi ikke har host. (Det skjer når du har lagret siden og ser på den siden uten at den er på en server. Da skaper google analytics masse feilmeldinger som fører til at det stopper opp.)